Deployment/Jenkins/agent/agent-instance.groovy

```diff
@@ -0,0 +1,258 @@
+                                },
+                                {
+                                    "ParameterKey": "PipRpm",
+                                    "ParameterValue": "${env.PipRpm}"
+                                },
+                                {
+                                    "ParameterKey": "PrivateIp",
+                                    "ParameterValue": "${env.PrivateIp}"
+                                },
+                                {
+                                    "ParameterKey": "ProvisionUser",
+                                    "ParameterValue": "${env.ProvisionUser}"
+                                },
+                                {
+                                    "ParameterKey": "PypiIndexUrl",
+                                    "ParameterValue": "${env.PypiIndexUrl}"
+                                },
+                                {
+                                    "ParameterKey": "PyStache",
+                                    "ParameterValue": "${env.PyStache}"
+                                },
+                                {
+                                    "ParameterKey": "SecurityGroupIds",
+                                    "ParameterValue": "${env.SecurityGroupIds}"
+                                },
+                                {
+                                    "ParameterKey": "SubnetId",
+                                    "ParameterValue": "${env.SubnetId}"
+                                },
+                                {
+                                    "ParameterKey": "ZapAgentUrl",
+                                    "ParameterValue": "${env.ZapAgentUrl}"
+                                }
+                            ]
+                        /
+                }
+            }
+        }
+        stage ('Prepare AWS Environment') {
+            options {
+                timeout(time: 1, unit: 'HOURS')
+            }
+            steps {
+                withCredentials(
+                    [
+                        [$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: "${AwsCred}", secretKeyVariable: 'AWS_SECRET_ACCESS_KEY'],
+                        sshUserPrivateKey(credentialsId: "${GitCred}", keyFileVariable: 'SSH_KEY_FILE', passphraseVariable: 'SSH_KEY_PASS', usernameVariable: 'SSH_KEY_USER')
+                    ]
+                ) {
+                    sh '''#!/bin/bash
+                        echo "Attempting to delete any active ${CfnStackRoot} stacks... "
+                        aws --region "${AwsRegion}" cloudformation delete-stack --stack-name "${CfnStackRoot}"
+
+                        aws cloudformation wait stack-delete-complete --stack-name ${CfnStackRoot} --region ${AwsRegion}
+                    '''
+                }
+            }
+        }
+        stage ('Launch Jenkins Master Stack') {
```

Comment:
Should this maybe be `Launch Jenkins Agent Stack`?

Reply:
YES